### PR TITLE
reorder staging workflow to let build-and-test read SSM parameters

### PR DESCRIPTION
**Pull request by vkapenkin**
Reordered staging workflow to let build-and-test step read SSM parameters

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -271,26 +271,28 @@ workflows:
...
+    jobs:
       - check-code-formatting:
           context: api-nuget-token-context
-      - build-and-test:
-          context:
-              - api-nuget-token-context
-              - SonarCloud
+      - assume-role-staging:
+          context: api-assume-role-housing-staging-context
+          requires:
```
Comment (humulla) [inline .circleci/config.yml:279]: The assume-role-staging and check-code-formatting jobs can run at the same time. Please remove lines 279 and 280.

**Comment by humulla** [comment]
Also, I would advise for you to wait till the certificate is regenerated as that may resolve the issue with the tests failing.

File: .circleci/config.yml (hunk 1 above)
Comment (vkapenkin) [reply .circleci/config.yml]: Removed.

There are two reasons for failing tests 
- no SSM permissions for build-and-test step on staging, this patch will fix it
- expired certificates

Anyway will wait till certificates are renewed.

**Comment by vkapenkin** [comment]
Test failing issue is resolved with the new pact broker certificate

**Review by humulla — APPROVED** [review]

**Pull request merged**